pkg/manager/manager.go

```diff
@@ -166,17 +173,79 @@ func (m *Manager) initializeStores() {
 
 }
 
+// Creates gRPC server and registers various services; then serves.
+func (m *Manager) startNBServers() error {
+	authorization := false
+	if oidcURL := os.Getenv(OIDCServerURL); oidcURL != "" {
+		authorization = true
+		log.Infof("Authorization enabled. %s=%s", OIDCServerURL, oidcURL)
+		// OIDCServerURL is also referenced in jwt.go (from onos-lib-go)
+		// and in gNMI Get() where it drives OPA lookup
+	} else {
+		log.Infof("Authorization not enabled %s", os.Getenv(OIDCServerURL))
+	}
+
+	s := northbound.NewServer(northbound.NewServerCfg(mgr.Config.CAPath, mgr.Config.KeyPath, mgr.Config.CertPath,
+		int16(mgr.Config.GRPCPort), authorization,
+		northbound.SecurityConfig{
+			AuthenticationEnabled: authorization,
+			AuthorizationEnabled:  authorization,
+		}))
+
+	s.AddService(logging.Service{})
+
+	adminService := admin.NewService(mgr.NetworkChangesStore, mgr.NetworkSnapshotStore, mgr.DeviceSnapshotStore)
+	s.AddService(adminService)
+
+	diagService := diags.NewService(mgr.DeviceChangesStore,
+		mgr.DeviceCache,
```

Comment:
Since the code is moving toward injecting deps, we should remove instances in the manager datas structure like this https://github.com/onosproject/onos-e2t/blob/master/pkg/manager/manager.go#L67. We don't need to define store instances in the manager structure and the instance can be injected to any package from Start function.

Reply:
Let's do this once we figure out how to get the tests working again.

Reply:
Sure.